trino/src/main/java/com/netease/arctic/trino/ArcticConnectorMetadata.java

```diff
@@ -81,6 +82,8 @@ public class ArcticConnectorMetadata implements ConnectorMetadata {
 
   private final ArcticCatalog arcticCatalog;
 
+  private final Map<SchemaTableName, ArcticTable> tableCache = new ConcurrentHashMap<>();
+
   public ArcticConnectorMetadata(
```

Comment:
Since Cache has already been implemented in the core, is it necessary to re-implement caching in Trino?

Reply:
I think it's necessary because loading the mix table not only loads the iceberg table but also has some other operations and I'm not sure whether the impact is big or not.

Reply:
It just loads iceberg table.

Reply:
It just loads iceberg table.

Reply:
There are still some 'if' judgment and wrapper operations in both 'trino' and 'core'. And the default cache expiration in the caching catalog is 30s. Trino query will easily exceed 30 seconds. For me, I don’t want to go to AMS to set the expiration time every time.

Reply:
Maybe you can overwrite AMS configured catalog properties in Trino engine configs.

Reply:
The life cycles of these two caches should be different. If I set a long expiration time in the catalog properties on the trino side, then the engine-side cache that should have been destroyed when the trino "ConnectorMetadata" life cycle ends will now have to wait for the time to expire, which may not be reasonable. If the setting time is too small, cache utilization will decrease.